src/Makefile

```diff
@@ -0,0 +1,18 @@
+BINDIR = ${DESTDIR}/usr/bin
+CC = gcc
+PKG_CONFIG = pkg-config
+PKG_NAMES = ssu-sysinfo
+PKG_LIBS = $(shell $(PKG_CONFIG) --libs $(PKG_NAMES))
+PKG_CFLAGS = $(shell $(PKG_CONFIG) --cflags $(PKG_NAMES))
+LIBS += $(PKG_LIBS)
+CFLAGS += $(PKG_CFLAGS)
+
+c-sample: c-sample.c
+	$(CC) -o $@ $^ $(CFLAGS) $(LIBS)
+
+install:
+	mkdir -p $(BINDIR)
+	install -m755 c-sample $(BINDIR)
```

Comment:
You can just use install -Dm755 in this case no need for mkdir.

Reply:
That didn't work when I tried it earlier. I didn't want to spend time fighting it so I just used the easier way. Turns out `install` interpreted $(BINDIR) so that `bin` was a filename, i.e. it copied `c-sample` to a file called `bin` in `usr`. Changed to use `-t` option for target dir.